src/containers/App/App.test.js

```diff
@@ -29,25 +122,27 @@ describe('App', () => {
       overview:
         'The near future, a time when both hope and hardships drive humanity to look to the stars and beyond. While a mysterious phenomenon menaces to destroy life on planet Earth, astronaut Roy McBride undertakes a mission across the immensity of space and its many perils to uncover the truth about a lost expedition that decades before boldly faced emptiness and silence in search of the unknown.',
       average_rating: 7
-    },
-    {
-      id: 3,
-      title: 'Frozen II',
-      poster_path:
-        'https://image.tmdb.org/t/p/original//pjeMs3yqRmFL3giJy4PMXWZTTPa.jpg',
-      backdrop_path:
-        'https://image.tmdb.org/t/p/original//xJWPZIYOEFIjZpBL7SVBGnzRYXp.jpg',
-      release_date: '2019-11-20',
-      overview:
-        'Elsa, Anna, Kristoff and Olaf head far into the forest to learn the truth about an ancient mystery of their kingdom.',
-      average_rating: 8
-    }
-  ];
+    }];
+    expect(mockAddMovies).toHaveBeenCalledWith(mockMovieArg);
+  })
 
-  it('should match the App Snapshot', () => {
-    wrapper = shallow(<App />);
-    expect(wrapper).toMatchSnapshot();
-  });
+  it('should invoke isLoading prop when fetchAllMovies resolves', () => {
+    expect(mockIsLoading).toHaveBeenCalledWith(false);
+  })
+
+  it('should invoke handleError prop if fetchAllMovies rejects', async () => {
+    fetchAllMovies.mockImplementation(() => {
+      return Promise.reject(Error('error '))
+    });
+    wrapper = await shallow(<App
+      allMovies={mockMoviesData}
+      addMovies={mockAddMovies}
+      handleError={mockHandleError}
+      isLoading={mockIsLoading}
+      />);
+    await wrapper.instance().forceUpdate();
+    expect(mockHandleError).toHaveBeenCalledWith('Data retrieval error - Please refresh the page');
+  })
```

Comment:
Very helpful for me to see a use-case for forceUpdate() as well as the use of async/await in a test context. Thanks, Heather